notifications_api_common/admin.py

```diff
@@ -9,7 +11,14 @@
 
 @admin.register(NotificationsConfig)
 class NotificationsConfigAdmin(SingletonModelAdmin):
-    pass
+    def save_model(self, request, obj, form, change):
+        if not getattr(settings, "SITE_DOMAIN", None):
+            raise ValidationError(
+                _(
+                    "SITE_DOMAIN must be set in settings before saving NotificationsConfig."
```

Comment:
It is true that it is a setting, but for the devops it will be a bit clearer if we mention this is an envvar
```suggestion
                    "The SITE_DOMAIN environment variable must be set before saving NotificationsConfig."
```